action.yml

```diff
@@ -23,6 +23,10 @@ inputs:
   version-tag:
     description: 'The version tag sequence to validate. Example: "v1.2.3".'
     required: true
+  should-exist:
+    description: 'Flag to indicate if the version tag should exist in the repository. Setting by `true` disables increment validity check. Default: "false".'
```

Comment:
```suggestion
    description: 'Flag to indicate if the version tag should exist in the repository. Setting to `true` disables increment validity check. Default: "false".'
```

Reply:
What does the action then do, if this flag is set to `true`?

Reply:
I hope I have answered in new table added in commit - 7012d8240648d44c8e28475e8c7fde4e3e6749dc.

Reply:
Another improvement can be seen in commit - 66aff399f2d6faf821aa22cf34ece4f8b9b6bf56.